### PR TITLE
Update HUD.lua

### PR DESCRIPTION
**Pull request by ModoSN**
Removing duplicate SetTextDropshadow

### CODE REVIEW

File: library/natives/GTAV/HUD.lua [hunk 1]
```diff
@@ -4295,11 +4295,6 @@ function SetTextColour(red, green, blue, alpha) end
 ---@param a number
 function SetTextDropshadow(distance, r, g, b, a) end
```
Comment (ModoSN) [inline library/natives/GTAV/HUD.lua:4296]: Exists here

**Comment by jag3dagster** [comment]
These are automatically generated files and the native you called out does in fact have two separate instances.

**Comment by ModoSN** [comment]
> These are automatically generated files and the native you called out does in fact have two separate instances.

What are they generated from and how often are they updated? I will submit PR to the FiveM docs

**Comment by thelindat** [comment]
Regardless, these are two different natives - `SET_TEXT_DROPSHADOW` vs `SET_TEXT_DROP_SHADOW`.

**Pull request closed without merging**